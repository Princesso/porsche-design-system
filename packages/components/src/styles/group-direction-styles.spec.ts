import { getGroupDirectionJssStyles } from './group-direction-styles';

describe('getGroupDirectionStyles()', () => {
  it.each<Parameters<typeof getGroupDirectionJssStyles>>([
    ['column'],
    ['row'],
    [{ base: 'column', xs: 'row', s: 'column', m: 'row', l: 'column', xl: 'row' }],
  ])('should return correct css for direction: %s', (...args) => {
    expect(getGroupDirectionJssStyles(...args)).toMatchSnapshot();
  });
});
