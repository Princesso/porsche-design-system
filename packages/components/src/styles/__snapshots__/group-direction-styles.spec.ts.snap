// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getGroupDirectionStyles() should return correct css for direction: {
  base: 'column',
  xs: 'row',
  s: 'column',
  m: 'row',
  l: 'column',
  xl: 'row'
} 1`] = `undefined`;

exports[`getGroupDirectionStyles() should return correct css for direction: column 1`] = `
{
  "alignItems": "stretch",
  "flexFlow": "column nowrap",
}
`;

exports[`getGroupDirectionStyles() should return correct css for direction: row 1`] = `
{
  "alignItems": "center",
  "flexFlow": "row wrap",
}
`;
